config/src/test/java/org/springframework/security/config/annotation/web/configurers/SessionManagementConfigurerTests.java

```diff
@@ -625,6 +702,44 @@ UserDetailsService userDetailsService() {
 
 	}
 
+	@Configuration
+	@EnableWebSecurity
+	static class ConcurrencyControlWithSessionLimitStrategyConfig {
+
+		@Bean
+		SecurityFilterChain filterChain(HttpSecurity http, SessionLimitStrategy sessionLimitStrategy) throws Exception {
+			// @formatter:off
+			http
+					.formLogin(withDefaults())
+					.sessionManagement((sessionManagement) ->
```

Comment:
We are moving towards the following syntax:

```java
http
    // ...
    .method((elements) -> elements
        .submethod((sub) -> sub
           // ...
        )
    )
```

That is, the lambda variable name and its usage go on the same line and the end parenthesis goes on the next line, like an end curly brace. Can you update this to use that format please?